src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/list/AdminAreasOfLawEndpointTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.dts.fact.admin.list;
+
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.dts.fact.model.admin.AreaOfLaw;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.BEARER;
+
+public class AdminAreasOfLawEndpointTest extends AdminFunctionalTestBase {
+
+    private static final String ADMIN_AREAS_OF_LAW_ENDPOINT = "/admin/courtAreasOfLaw";
+
+    @Test
+    public void shouldGetAllCourtAreasOfLaw() {
```

Comment:
@ashwini-mv the method name needs to be updated. This is not getting areas of law for a specific court so the method name should be "shouldGetAllAreasOfLaw"